h_transport_materials/references.bib

```diff
@@ -2251,68 +2251,112 @@ @article{fowler_tritium_1977
+	language = {en},
+	urldate = {2023-05-08},
+	journal = {Journal of Nuclear Materials},
+	author = {Chen, Ze and Hu, Xunxiang and Ye, Minyou and Wirth, Brian D.},
+	month = jun,
+	year = {2021},
 	keywords = {Deuterium retention, Deuterium transport, Reduced activation ferritic-martensitic steels},
-	abstract = {Reduced activation ferritic-martensitic (RAFM) steels have been developed for decades for use as fusion blanket structural materials, and have advantages in both mechanical properties and irradiation resistance following careful engineering of the microstructure. However, the hydrogen isotope behavior in these proposed fusion structural materials is not well understood, but is important to assess since it impacts the fusion reactor safety and self-sufficient tritium fuel cycle. In this work, we investigated deuterium transport and retention in representative advanced RAFM steels, including castable nanostructured alloys (CNAs), and oxide-dispersion-strengthened (ODS) steels. A gas-driven permeation (GDP) system was used to measure the permeability, diffusivity and solubility of the studied materials, covering the temperature range from 623 K to 873 K, and the loading pressures from 1.8x104 to 1.0x105 Pa. The results indicated that the deuterium permeability has little material dependence. In contrast, the deuterium diffusivity of the studied materials showed significant variation. The deuterium diffusivity in ODS steels is one order of magnitude lower than that in RAFM steels and CNAs, and correspondingly, have an effective solubility that is 2-10 times larger than RAFM steels and CNAs. In addition, thermal desorption spectroscopy (TDS) measurements were performed to assess the deuterium retention and desorption of these materials following a static thermal deuterium charging at 723 Kfor 1 hour under the deuterium pressure of 1.0x105Pa. It was found that ODS steels exhibit the highest deuterium retention and have broader desorption peaks. Microstructural features contributing to deuterium retention and impacting deuterium transport are discussed to rationalize the observed deuterium behavior in the studied RAFM steels.}
+	pages = {152904},
+	file = {ScienceDirect Snapshot:C\:\\Users\\remidm\\Zotero\\storage\\X56J64CD\\S0022311521001288.html:text/html},
```

Comment:
Think we need to remove these file objects from the references really, but could do them all and create a standardised reference method in another PR